1MI0800224/week2/week2_task2.cpp

```diff
@@ -0,0 +1,39 @@
+#include <iostream>
+#include <fstream>
+using namespace std;
+
+bool IsSymbolAorD(const char symbol){

```

Comment:
This can be re-written as `IsOneOf` and with the help of [this one](https://en.cppreference.com/w/cpp/utility/variadic). So you'll have a function that you can use whether `symbol` is equal to one of bunch that are "wanted" :P